### PR TITLE
HTTP 500 alarm and webhook

### PR DESCRIPTION
**Pull request by mbarton**
This PR updates the cloud formation JSON to include an alarm on HTTP 500 responses which routes through to a webhook via SNS.

When deployed with the right parameters, this hooks up media-atom-maker to pager duty.

### CODE REVIEW

File: cloudformation/media-atom-maker.json [hunk 1]
```diff
@@ -55,6 +55,14 @@
         "AuditTable": {
             "Description": "Name of the audit dynamo table",
             "Type": "String"
+        },
+        "AlertWebhook": {
+          "Description": "Where CloudWatch alerts are sent",
+          "Type": "String"
+        },
+        "AlertActive": {
```
Comment (akash1810) [inline cloudformation/media-atom-maker.json]: when would we make this false?

Comment (mbarton) [inline cloudformation/media-atom-maker.json]: I don't know. I was copying how the grid does it.

Happy to remove it if you want?

Comment (akash1810) [inline cloudformation/media-atom-maker.json]: ah ok. It'll probably be best to keep all alarms on for now. We can revisit later once the app has proven stability.

Comment (akash1810) [inline cloudformation/media-atom-maker.json]: is an alarm that can be turned off, worthy of being an alarm?

if a tree falls in the forest...

Comment (mbarton) [inline cloudformation/media-atom-maker.json]: to avoid the philosophical debate, I've removed it :)

**Pull request merged**